website/docs/d/cloud_provider_access.markdown

```diff
@@ -6,6 +6,8 @@ description: |-
     Allows you to get the list of cloud provider access roles
 ---
 
+**WARNING:** The data source `mongodbatlas_cloud_provider_access` is deprecated and will be removed in version v1.14.0, use the data source `mongodbatlas_cloud_provider_access_setup` instead.
```

Comment:
have my doubts here, want to be sure that data source is the correct replacement for the mongodbatlas_cloud_provider_access data source

Reply:
it is not the perfect replacements as some fields are differents but it is the closest